addon/components/pix-checkbox.hbs

```diff
@@ -1,6 +1,5 @@
-<label class="pix-checkbox {{@class}}" for={{@id}}>
```

Comment:
Attention niveau accessibilité : https://www.w3.org/WAI/tutorials/forms/labels/

`. Generally, explicit labels are better supported by assistive technology.`

Reply:
garder le for / id . mais avec un id auto généré par le composant ?

Reply:
En fait, si on sépare l'`input` du `label`, à ce moment là il faut utiliser les attributs `id` et `for`. Mais lorsque le label englobe le l'input, ce n'est pas nécessaire :)

Reply:
Les deux sont ok niveau a11y

Reply:
cf https://www.w3.org/WAI/tutorials/forms/labels/#associating-labels-implicitly

Reply:
Oui c'est pas nécessaire lorsque l'on wrappe je suis ok. De ce que je comprend c'est compliant, ça marche. Mais certaines techo d'aide prèfere que ce soit défini explicitement. d'où l'exxtrait  `Generally, explicit labels are better supported by assistive technology.`

Reply:
yes je suis d'accord avec toi, on est pas allé jusqu'à ce point en refacto :)

Reply:
C'est quelque chose qui était déjà présent avant, on l'enlève ici 🤔 Il faudrait ajouter un computed `id` à la place de l'argument `id` dont l'implémentation utiliserait guidFor pour générer un identifiant unique 👍

Reply:
C'est modifié